.github/workflows/test.yml

```diff
@@ -78,11 +78,18 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [["3.10", "310"], ["3.11", "311"]]
+        python-version: [["3.11", "311"]]
         # run in modes: --optimize [gas, none, codesize]
-        flag: ["core", "no-opt", "codesize"]
+        opt-mode: ["gas", "none", "codesize"]
```

Comment:
i don't think we should omit the argument since we want to make sure which modes are being tested (in case we ever change the default). but we could have an explicit test for which is the default setting

Reply:
documenting from offline discussion. This would work:
```console
vyper MyContract.vy == vyper --optimize gas MyContract.vy
```

Reply:
default settings test added in https://github.com/vyperlang/vyper/pull/3496/commits/ffeb8d61d0f6d7e5e43b7fc3cca8a2674c181a7a